tests/test.ts

```diff
@@ -28,7 +29,12 @@ describe('Test help for each function', () => {
   });
 });
 
-describe.skip('Test clasp list function', () => {
+describe('Test clasp list function', () => {
+  before(function() {
```

Comment:
Can you extract this to a function?


function skipIfPR() {

Reply:
Not quite sure this will work, Typescript is not a fan:

```
[ts] 'this' implicitly has type 'any' because it does not have a type annotation.
```

I'm not sure the best way to pass `this` through that makes it work.

Reply:
Also, `isPR` is never a boolean, it's either an `int` like this PR is `246` or it is `"false"` (string)

Reply:
OK, we can follow up in a different change.